core/src/main/java/com/backblaze/b2/client/B2StorageClientWebifierImpl.java

```diff
@@ -425,12 +425,11 @@ public B2FileVersion getFileInfo(B2AccountAuthorization accountAuth,
     }
 
     @Override
-    public B2FileVersion getFileInfoByName(B2AccountAuthorization accountAuth, B2GetFileInfoByNameRequest request) throws B2Exception {
-        B2HeadersImpl.Builder builder = B2HeadersImpl.builder();
-        webApiClient.head(makeGetFileInfoByNameUrl(accountAuth, request.getBucketName(), request.getFileName()),
-                makeHeaders(accountAuth)).forEach((key, value) -> builder.set(key, value));
+    public B2FileVersion getFileInfoByName(B2AccountAuthorization accountAuth,
+                                           B2GetFileInfoByNameRequest request) throws B2Exception {
+        B2Headers headers = webApiClient.head(makeGetFileInfoByNameUrl(accountAuth, request.getBucketName(),
+                request.getFileName()), makeHeaders(accountAuth));
 
-        B2Headers headers = builder.build();
 
         return new B2FileVersion(headers.getValueOrNull(FILE_ID), headers.getValueOrNull(FILE_NAME),
                 headers.getContentLength(), headers.getContentType(), headers.getContentSha1OrNull(),
```

Comment:
please pass "upload" as the action here, instead of null (on line 436).

Reply:
ok